src/common/contentPreference.ts

```diff
@@ -516,3 +518,28 @@ export const unblockEntity = async ({
   // currently unblock is just like unfollow, eg. remove everything from db
   return unfollowEntity({ ctx, id, entity, feedId });
 };
+
+export const whereNotUserBlocked = (
+  qb: QueryBuilder,
+  {
+    userId,
+    feedId,
+  }: Partial<{
+    userId: string;
+    feedId: string;
+  }>,
+) => {
+  const feedIds = feedId ? [feedId, userId] : [userId];
```

Comment:
```suggestion
    userId: string;
    feedId?: string;
  }>,
) => {
  const feedIds = feedId ? [feedId, userId] : [userId];
```

if feedId can be optional this should be added in the function's type